package-lock.json

```diff
@@ -1,61 +1,62 @@
 {
   "name": "openid-federation-browser",
-  "version": "0.11.0",
+  "version": "0.13.0",
```

Comment:
Since we have in production 0.14.0

I suppose to have here 0.15.0

<img width="577" height="263" alt="image" src="https://github.com/user-attachments/assets/d2f617be-6f7e-4b26-a059-646e8d9bc62f" />